src/JSONDB.php

```diff
@@ -100,7 +107,7 @@ private function check_file()
         } else {
             // Read content of JSON file
             $content = file_get_contents($this->file);
-            $content = json_decode($content, true);
+            $content = json_decode( ($content=="")?'[]':$content, true );
```

Comment:
I do not think this is relevant. We require a JSON file, not an empty text file. So I believe we have to let the user know that this is an invalid JSON file than assume it is supposed to be one.

Reply:
[English by google traslator]ok, i think i thought to fix a problem in case the "json" was not validated, which was not necessary.

but at the same time i thought to fix the creation of the file if the json was not valid, that was my main intention, to create the json file without problems.

Reply:
Yes. Your fix is good, but I think it is overengineered in this library. It has to be JSON file content to be read by the library.
An empty file `""` is an invalid JSON. An empty JSON file `[]` is an empty JSON

I hope this makes sense.

Reply:
You can revert this line so I can merge.

Reply:
of course